data/README.rst

```diff
@@ -0,0 +1,107 @@
+Software Directory README
```

Comment:
I'm browsing the website source using various tools (pretty much whatever I have handy at the time.) Although I'm not sure if those tools will bork over the extension, why run the risk? Using a commonly used extension will prevent an issue, if there's one. 

If you do not agree, I wil happily shrug and move on. To me, it's a trivial issue at best. :)